hooks/conf_regen/15-nginx

```diff
@@ -89,25 +90,21 @@ do_pre_regen() {
     export redirect_to_https="$(jq -r '.nginx_redirect_to_https' <<< "$YNH_SETTINGS" | int_to_bool)"
     export compatibility="$(jq -r '.nginx_compatibility' <<< "$YNH_SETTINGS" | int_to_bool)"
     export experimental="$(jq -r '.security_experimental_enabled' <<< "$YNH_SETTINGS" | int_to_bool)"
-    export tls_passthrough_enabled="$(jq -r '.tls_passthrough_enabled' <<< "$YNH_SETTINGS" | int_to_bool)"
-    export tls_passthrough_list="$(jq -r '.tls_passthrough_list' <<< "$YNH_SETTINGS" | int_to_bool)"
+    export tls_passthrough_list="$(jq -r '.tls_passthrough_list' <<< "$YNH_SETTINGS" | sed 's/^null$//g')"
+    export trusted_proxies_list="$(jq -r '.trusted_proxies_list' <<< "$YNH_SETTINGS" | sed 's/^null$//g')"
 
     do_base_regen "${pending_dir}"
 
     local tls_passthrough_module="${pending_dir}/etc/nginx/modules-enabled/tls_passthrough.conf"
     mkdir -p "${pending_dir}/etc/nginx/modules-enabled/"
 
-    if [[ "$tls_passthrough_enabled" == "True" ]]; then
-        ynh_render_template "tls_passthrough.conf" "${tls_passthrough_module}"
-        for tls_passthrough_domain_and_ip in ${tls_passthrough_list//,/ }; do
-            export tls_passthrough_domain=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $1}')
-            export tls_passthrough_ip=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $2}')
-            export tls_passthrough_port=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $3}')
-            ynh_render_template "tls_passthrough_server.conf" "${nginx_conf_dir}/${tls_passthrough_domain}.forward80.conf"
-        done
-    else
-        touch "${tls_passthrough_module}"
-    fi
+    ynh_render_template "tls_passthrough.conf" "${tls_passthrough_module}"
+    for tls_passthrough_domain_and_ip in ${tls_passthrough_list//,/ }; do
+        export tls_passthrough_domain=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $1}')
+        export tls_passthrough_ip=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $2}')
+        export tls_passthrough_port=$(echo "$tls_passthrough_domain_and_ip" | awk -F';' '{print $3}')
```

Comment:
```suggestion
        export tls_passthrough_domain=$(awk -F';' '{print $1}' <<< "$tls_passthrough_domain_and_ip")
        export tls_passthrough_ip=$(awk -F';' '{print $2}' <<< "$tls_passthrough_domain_and_ip")
        export tls_passthrough_port=$(awk -F';' '{print $3}' <<< "$tls_passthrough_domain_and_ip")
```

Nitpicking here though I may be the one that wrote the original code, but using `<<<` instead of `echo` prevents the content from actually ending up in the logs which in turn should prevent it to leak when sharing logs (though we have other ways to anonymize but maybe only the domain names "actually registered" are redacted)